website/docs/foundations/colors/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,99 @@
+
+In rare instances where token usage is necessary outside of the context of a component, following these standards ensures consistency across our UI:
+- `Foreground / Disabled` for disabled text, icons or other foreground elements
+- `Border / Primary` for disabled borders
+- `Surface / Interactive-Disabled` for disabled backgrounds on interactive elements
+
+![Image showing a disabled input with all the associated semantic tokens](/assets/foundations/color/colors-disabled-examples.png)
+
+## Accessible color combinations
+
+We intend to be conformant with WCAG 2.2 Level AA requirements. In terms of color contrast, this means a luminosity ratio of 4.5:1 for normal sized text, and 3:1 for large text. Further details are outlined on [WCAG’s understanding of Contrast (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html). By default, semantic color tokens provide accessible color combinations out of the box with their associated naming conventions, while using palette colors requires manual validation, especially if you plan to mix and match. As an example, if a color has a semantic status name in it, then other associated status colors will be accessible.
+
+### Using semantic tokens
+
+Semantic colors were designed to be used together, ensuring they meet accessibility standards and look visually pleasing. For example, foreground colors are intended to be used with surface or page colors. In many cases, it is important to use specific contextual naming conventions, such as status colors, together. Here are some examples of semantic color combinations with their respective contrast ratios:
+- `Foreground / Strong` on `Surface / Primary` nets a ratio of 19.54:1
+- `Foreground / Success-on Surface` on `Surface / Success` nets a ratio of 5.37:1
+- `Foreground / Primary` on `Surface / Primary` nets a ratio of 10.82:1
+- `Foreground / Action` on `Surface / Faint` nets a ratio of 4.86:1
+
+![Image showing color ratios of a card with details inside of it](/assets/foundations/color/colors-semantic-tokens-accessibility-examples.png)
+
+### Using palette colors
+
+If semantic colors do not meet your needs, a set of base palette colors are also available. These styles are usage-agnostic, which can introduce challenges in consistently scaling a complex design but allow more freedom and flexibility in color pairing. Keep in mind when pairing colors on your own, accessible contrast ratios between adjacent colors must meet their respective values. You can use free tools like the [WebAIM contrast checker](https://webaim.org/resources/contrastchecker/) or [Stark’s Figma plugin](https://www.figma.com/community/plugin/732603254453395948/stark-contrast-accessibility-checker) to validate your color combination choices.
```

Comment:
While these nods to accessibility, the information is more general than that. Would it be more valuable to move them under the "What are semantic colors?" section and bring them to an H2?

Reply:
Not a bad idea, I can move it.